docs/source/components/nodes/stereo_depth.rst

```diff
@@ -132,7 +238,7 @@ However this distance can be cut in 1/2 (to around 35cm for the OAK-D) with the
 
 2. Enabling Extended Disparity.
 
-This mode extends the disparity search to a maximum of 191 pixels from the standard 96 pixels, thereby 1/2-ing the minimum depth.
+Extended Disparity mode increases the levels of disparity to 191 from the standard 96 pixels, thereby 1/2-ing the minimum depth. It does so by computing the 96-pixel disparities on the original 1280x720 and on the downscaled 640x360 image, which are then merged to a 191-level disparity. The reason there are only 191 levels of disparity is because the value 0 is always common between the two.
```

Comment:
> The reason there are only 191 levels of disparity is because the value 0 is always common between the two.
Remove this part. I just said it to explain you guys.